variables.tf

```
@@ -58,3 +58,9 @@ variable "enable_s3_public_access_block" {
   type        = bool
   default     = true
 }
+
+variable "point_in_time_recovery" {
```

To make it clear we are enabling point in time recovery for dynamodb
```suggestion 
variable "dynamodb_point_in_time_recovery" {
```